pyiron/atomistics/structure/atoms.py

```diff
@@ -1357,45 +1334,24 @@ def get_neighbors(
             tolerance (int): tolerance (round decimal points) used for computing neighbor shells
             id_list (list): list of atoms the neighbors are to be looked for
             cutoff_radius (float): Upper bound of the distance to which the search must be done
-            boundary_width_factor (float): width of the layer to be added to account for pbc.
+            width_buffer (float): width of the layer to be added to account for pbc.
```

Comment:
Also misses a docs for some of the arguments, should also be in the same order as the argument list.